### PR TITLE
Fix cropped icon on windows

### LINKED ISSUE #1236: Workbench desktop icon is cropped/squashed
After replacing the interim icon with our standard `InVEST.ico` icon, the icon - as it appears in the Windows tray, or in the upper-left corner of the Workbench window - is not even close to square.

### PR DESCRIPTION
**Pull request by emlys**
## Description
Fixes #1236 
It turns out that both operating systems are very picky about icon sizes. Using a 256x256 icon resolved the issue on windows, but then the mac build failed with `ERR_ICON_TOO_SMALL`. So now we have separate icons for each.

## Checklist
- [ ] Updated HISTORY.rst and link to any relevant issue (if these changes are user-facing)
- [ ] Updated the user's guide (if needed)
- [ ] Tested the affected models' UIs (if relevant)

### CODE REVIEW

File: workbench/electron-builder-config.js [hunk 1]
```diff
@@ -42,12 +42,12 @@ const config = {
   artifactName: ARTIFACT_NAME,
   mac: {
     category: 'public.app-category.business',
-    icon: 'resources/InVEST-2.ico',
+    icon: 'resources/InVEST-2-574x574.ico',
```
Comment (dcdenu4) [inline workbench/electron-builder-config.js:45]: Woah, 574 is a fun dimension size. Was 512x512 too small as well or was there a particular reason for going 574x574?

**Review by dcdenu4 — COMMENTED** [review]
Hey @emlys, I just had a quick question about the chosen size for the mac icon.

File: workbench/electron-builder-config.js (hunk 1 above)
Comment (emlys) [reply workbench/electron-builder-config.js:45]: 574x574 was the size of the original image before this PR, so I just stuck with that.

**Review by dcdenu4 — APPROVED** [review]
Looks good Emily! Sorry random @ e person!

**Pull request merged**